mlflow/protos/service.proto

```diff
@@ -406,6 +406,27 @@ service MlflowService {
     };
   }
 
+   // Log a param used for a run. A param is a key-value pair (string key,
+  // string value). Examples include hyperparameters used for ML model training and
+  // constant dates and values used in an ETL pipeline. A param can be logged only once for a run.
+  //
+  rpc updateArtifactsLocation(UpdateArtifactsLocation) returns (UpdateArtifactsLocation.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
```

Comment:
Since we update an already existing field, the method sould be PUT

Reply:
Done